cli/cmd/cmdutil/cmdutil.go

```diff
@@ -73,6 +74,18 @@ func TextPrinter(str string) {
 	boldGreen.Fprintln(color.Output, str)
 }
 
+// Create admin client
+func Client(cfg *config.Config) (*client.Client, error) {
+	userAgent := fmt.Sprintf("rill-cli/%s", cfg.Version.String())
```

Comment:
When building from source, `cfg.Version` is `""`. In these cases, it should put `rill-cli/unknown`.

Reply:
if we put `unknown` then how will check if version is lower or not? ( i mean how will check for [https://github.com/rilldata/rill-developer/issues/1960](https://github.com/rilldata/rill-developer/issues/1960) ?)

Reply:
Btw i checked that, in case of build from source, its returning as `user-agent:[rill-cli/unknown (built from source) grpc-go/1.49.0]]`